api/src/main/data/xlit_scheme-utf8.xml

```diff
@@ -384,6 +384,13 @@
 <doc>ISO 11940-2 is an ISO standard for the romanization of the Thai alphabet almost identical to RTGS, published in 2007. https://en.wikipedia.org/wiki/ISO_11940-2</doc>
 </scheme>
 <scheme>
+<numeric>315</numeric>
+<name>MLCTS</name>
```

Comment:
Make this lowercase, seems to be the naming convention for other names in the file